besu/src/test/resources/everything_config.toml

```diff
@@ -192,6 +192,8 @@ tx-pool-retention-hours=999
 tx-pool-max-size=1234
 tx-pool-limit-by-account-percentage=0.017
 tx-pool-min-gas-price=1000
+tx-pool-allow-list=["0xABC0000000000000000000000000000000001234"]
+tx-pool-reject-list=["0xDEF0000000000000000000000000000000001234"]
```

Comment:
won't this cause an error if both are specified in the toml file?

Reply:
I did not seem to cause an issue, but can double check